src/sentry/api/urls.py

```diff
@@ -1906,6 +1907,11 @@
             ]
         ),
     ),
+    re_path(
+        r"^(?P<organization_slug>[^/]+)/metrics/code-locations/$",
```

Comment:
Separate from the previous comment, metrics meta data is a DDM-specific feature, while `/:org/meta/` is the generic metrics API (as per metrics platform). Since this is an experimental endpoint, we can postpone a final decision on its placement, but we should be clear that this is a feature-specific endpoint.